Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/AutoMoveTo.java

```diff
@@ -1,36 +1,61 @@
 package org.usfirst.frc.team199.Robot2018.commands;
 
-import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team199.Robot2018.Robot;
+import org.usfirst.frc.team199.Robot2018.autonomous.AutoUtils;
+import org.usfirst.frc.team199.Robot2018.autonomous.PIDSourceAverage;
+
+//import org.usfirst.frc.team199.Robot2018.subsystems.Drivetrain;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
 
 /**
  *
  */
-public class AutoMoveTo extends Command {
-
-    public AutoMoveTo(String[] points) {
-        // Use requires() here to declare subsystem dependencies
-        // eg. requires(chassis);
-    }
-
-    // Called just before this Command runs the first time
-    protected void initialize() {
-    }
-
-    // Called repeatedly when this Command is scheduled to run
-    protected void execute() {
-    }
-
-    // Make this return true when this Command no longer needs to run execute()
-    protected boolean isFinished() {
-        return false;
-    }
-
-    // Called once after isFinished returns true
-    protected void end() {
-    }
-
-    // Called when another command which requires one or more of the same
-    // subsystems is scheduled to run
-    protected void interrupted() {
+public class AutoMoveTo extends CommandGroup {
+	
+    public AutoMoveTo(String[] args) {
+        //requires(Drivetrain);
+    	double rotation;
+    	double[] point = new double[2];
+    	String parentheseless;
+    	String[] pointparts;
+    	for (String arg : args) {
+    		if (AutoUtils.isDouble(arg)) {
+    			rotation = Double.valueOf(arg);
+    			addSequential(new PIDTurn(rotation - AutoUtils.getRot(), Robot.dt, Robot.dt.getGyro()));
+    			AutoUtils.setRot(rotation);
+    		} else if (AutoUtils.isPoint(arg)) {
+    			parentheseless = arg.substring(1, arg.length() - 1);
+    			pointparts = parentheseless.split(",");
```

Comment:
put in a way to check/account for an accidental space after the comma in your scripts because it's probably going to happen at least once and mess everything up